src/server/job.rs

```diff
@@ -107,6 +112,12 @@ pub struct Job {
 
     pub submission_date: DateTime<Utc>,
     pub completion_date: Option<DateTime<Utc>>,
+
+    /// Holds channels that will receive information about the job task counters after the job
+    /// finishes in any way.
+    ///
+    /// You can subscribe to the finish updates with [`Self::subscribe_to_completion`].
+    completion_handlers: Vec<oneshot::Sender<JobTaskCounters>>,
```

Comment:
Why handlers? Isnt it raher "callbacks" or "senders"?

Reply:
Good point, `callback` is a better name here.